lib/ws_servers/api/algos/algo_worker.js

```diff
@@ -0,0 +1,188 @@
+'use strict'
+
+const debug = require('debug')
+const d = debug('bfx:hf:server:algo-worker')
+
+const { AOHost } = require('bfx-hf-algo')
+
+const validateAO = require('./util/validate_ao')
+
+class AlgoWorker {
+  constructor (settings, algoOrders, bcast, algoDB) {
+    this.host = null
+    this.userID = null
+
+    this.settings = settings
+    this.algoOrders = algoOrders
+
+    this.pub = bcast.ws
+
+    this.algoDB = algoDB
+  }
+
+  async start (apiKey, apiSecret, userID) {
+    const { dms, affiliateCode, wsURL, restURL } = this.settings
+
+    this.userID = userID
+
+    d(
+      'spawning bfx algo host (dms %s) [aff %s]',
+      dms ? 'enabled' : 'disabled',
+      affiliateCode
+    )
+
+    const wsSettings = {
+      apiKey,
+      apiSecret,
+      dms: dms ? 4 : 0,
```

Comment:
this magic number is duplicated in the code, what about moving to constant with a better description

Reply:
fixing for this occurency now, lets refactor this afterwards on the whole codebase, creating a follow up task to not bloat the changeset

Reply:
fixed!